cmd/policy/policy.go

```diff
@@ -163,13 +163,23 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 	}()
 
 	logs := func() *cobra.Command {
-		var after, before, outputFile, ownerID, context string
+		var after, before, outputFile, ownerID, context, decisionID string
+		var policyBundle bool
 		var request policy.DecisionQueryRequest
 
 		cmd := &cobra.Command{
-			Short: "Get policy decision logs",
-			Use:   "logs",
-			RunE: func(cmd *cobra.Command, _ []string) (err error) {
+			Short: "Get policy decision logs / Get decision log (or policy bundle) by decision ID",
+			Use:   "logs [decision_id]",
+			RunE: func(cmd *cobra.Command, args []string) (err error) {
+				if len(args) == 1 {
+					decisionID = args[0]
+				}
+				if decisionID != "" && (after != "" || before != "" || request.Status != "" || request.Offset != 0 || request.Branch != "" || request.ProjectID != "") {
+					return fmt.Errorf("filters are not accepted when decision_id is provided")
```

Comment:
I consider this to be almost a little too defensive. If the decision ID is present, I would just ignore the filters...

Either way is ok. Maybe get more than 1 opinion on this :)

Reply:
The real benefit of this check is the feedback it gives to the user, added defensiveness is just a bonus.
I would like to keep it for its documentation value to the user.